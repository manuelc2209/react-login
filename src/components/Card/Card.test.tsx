import React from 'react';
import renderer from 'react-test-renderer';
import { Card } from '.';

it('renders snapshot correctly', () => {
    const tree = renderer.create(<Card />).toJSON();
    expect(tree).toMatchSnapshot();
});
