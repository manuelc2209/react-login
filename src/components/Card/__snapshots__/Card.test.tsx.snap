// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<div
  className="sc-dkPtRN eGWjTe"
>
  <a
    className="sc-jRQBWg"
    target="_self"
  >
    <div
      className="sc-hKwDye"
    />
  </a>
  <div
    className="sc-eCImPb jbNsEY"
  />
</div>
`;
